packages/react-notion-x/src/components/eoi.tsx

```diff
@@ -11,27 +13,24 @@ export const EOI: React.FC<{
   inline?: boolean
   className?: string
 }> = ({ block, inline, className }) => {
-  const { components } = useNotionContext()
+  const { components, darkMode } = useNotionContext()
   const { original_url, attributes, domain } = block?.format || {}
-  if (!original_url || !attributes) {
+  if (!original_url) {
     return null
   }
 
-  const title = attributes.find((attr) => attr.id === 'title')?.values[0]
-  let owner = attributes.find((attr) => attr.id === 'owner')?.values[0]
-  const lastUpdatedAt = attributes.find((attr) => attr.id === 'updated_at')
-    ?.values[0]
-  const lastUpdated = lastUpdatedAt ? formatNotionDateTime(lastUpdatedAt) : null
   let externalImage: React.ReactNode
+  let owner: string = attributes?.find((attr) => attr.id === 'owner')?.values[0]
+  let title: string = attributes?.find((attr) => attr.id === 'title')?.values[0]
 
   switch (domain) {
-    case 'github.com':
-      externalImage = <SvgTypeGitHub />
-      if (owner) {
-        const parts = owner.split('/')
-        owner = parts[parts.length - 1]
-      }
+    case 'github.com': {
+      externalImage = <SvgTypeGitHub fill={darkMode ? '#fff' : '#161614'} />
```

Comment:
this fill doesn't have to rely on `darkMode`. I'd rather have it use the currentColor and have this logic be handled strictly in CSS